src/glyphs.js

```diff
@@ -111,6 +135,61 @@ const updateMessages = (id, path, req) => {
   )
 };
 
+const importMessages = (line) => {
+  const message = line.split(",");
+  const viewMessage = (message[1] === "Intro");
+
+  let path = "";
+  let newMessage = {"text": message[2]};
+
+  if (viewMessage) newMessage.probability = Number(message[3]);
+  else newMessage.probability = message.splice(3, 5).map(num => Number(num));
+
+  if (viewMessage) path = "view";
+  else path = `view.gauges.${message[1]}`;
+
+  return db.collection.updateOne(
+    {
+      "view.name": message[0].toLowerCase()
+    },
+    {
+      $addToSet: {
+        [`${path}.messages`]: newMessage
+      }
+    }
+  )
+}
+
+const overwriteMessages = (file) => {
+  let first = true;
+  let overwritten = [];
+
+  lineReader.eachLine(file, function(line) {
+    if (first) first = false;
+    else {
+      const message = line.split(",");
+      let path = "";
+
+      if (message[1] === "Intro") path = "view";
```

Comment:
I think both of these strings should probably be constants stored in variables somewhere.

Reply:
Maybe only "Intro"